src/system/OS_info/os_info_windows.cpp

```diff
@@ -80,10 +81,58 @@ static std::string version_name() {
 	return ret.substr(0, ret.find('|'));
 }
 
+unsigned int build_number() {
+	HKEY hkey{};
+	if(RegOpenKeyExA(HKEY_LOCAL_MACHINE, R"(Software\Microsoft\Windows NT\CurrentVersion)", 0, KEY_READ, &hkey))
+		return {};
+
+	DWORD ubr      = 0;
+	DWORD ubr_size = sizeof(ubr);
+	if(!RegQueryValueExA(hkey, "UBR", nullptr, nullptr, reinterpret_cast<LPBYTE>(&ubr), &ubr_size))
+		return ubr;
+
+	// Fall back to BuildLabEx in the early version of Windows 8.1 and less.
+	constexpr auto buildlabex_name = "BuildLabEx";
```

Comment:
inline this into the RQVEA calls

Reply:
It was to prevent from having duplicated strings and modifying only one of them by mistake, but yes, I will fix them :)